pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -315,13 +336,13 @@ def connection_list(context):
     """
     List the WBEM connection definitions.
 
-    List the WBEM connection definitions in the connections file as a table.
-
-    In the table, the current connection is marked with an "*" after its name.
+    This command displays all entries in the connections file and the
+    current connection if it is not in the connections file as
+    a table using the command line output_format to define the
+    table format.
```

Comment:
On "using the command line output_format to define the table format.":
1. Don't describe this using the name of an internal variable (output_format).
2. I don't think we need to repeat every time we say an output is a table that the format of the table can be controlled.

I suggest to drop the quoted part of the sentence.

Reply:
DONE